sdk/src/bitmap/utils.ts

```diff
@@ -0,0 +1,67 @@
+import {InvalidArraySizeError, InvalidBitMapValueError} from './errors';
+import {BigNumber} from 'ethers';
+
+/**
+ * Flips a specific bit in a `BigNumber` object and returns a new `BigNumber` object with the bit flipped.
+ *
+ * @export
+ * @param {number} index
+ * @param {BigNumber} num
+ * @return {*}  {BigNumber}
+ */
+export function flipBit(index: number, num: BigNumber): BigNumber {
+  const mask = BigNumber.from(1).shl(index & 0xff);
```

Comment:
Yep is working as that, we are using `0xff` because this is used for the failure map in the contracts and is an `uint256`. These functions are only used in the contracts once for the tests of the filure map. Given that they are only used once idk if want to complicate them further or start adding checks. Let me know what you would do

Reply:
My concern is someone using this not realising that it will modulo. I think I would prefer it error out above 256 bits than wrap around (see the whole drama of solidity <8 and unchecked integer addition.